templates/base/eslint.config.mjs

```diff
@@ -17,5 +17,16 @@ export default [
   pluginJs.configs.recommended,
   ...tseslint.configs.recommended,
   pluginReact.configs.flat.recommended,
-  eslintPluginPrettierRecommended,  
+  eslintPluginPrettierRecommended,
+  {
+    name: 'Custom Rules',
+    rules: {
+      'no-console': 2,
```

Comment:
Can we use the [string](https://eslint.org/docs/latest/use/configure/rules#rule-severities) syntax to set the rules? Strings explicitly name the severity ("error", "warn", "off"), so anyone reading the config sees immediately what is intended. Also reduces cognitive load: no need to memorize the mapping.